### PR TITLE
Add Card component

### PR DESCRIPTION
**Pull request by sumeetyeola**
-Screenshots
Screenshot (238)
Screenshot (239)
Screenshot (240)
Screenshot (241)
Screenshot (242)

### CODE REVIEW

File: components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,336 @@
+.btn {
+  border: none;
+  padding: 0.5rem 0.75rem;
+  font-family: inherit;
```
Comment (Ayushi20-19) [inline components/Card/card.css:4]: nice use

File: css/utilities.css [hunk 2]
```diff
@@ -15,6 +15,11 @@
   --warning-text: #ffedd5;
   --pink: #ff363c;
   --white: #fff;
+  --dark-green: #178344;
+  --net-price: #e1ecf1;
+  --net-price-border: #176c93;
+  --black-1: #111;
```
Comment (Ayushi20-19) [inline css/utilities.css:21]: suggestion: name can be improved..

**Review by Ayushi20-19 — APPROVED** [review]

**Review by VishalPatil18 — APPROVED** [review]
The code looks quite clean. Great work @sumeetyeola 👍

File: css/utilities.css (hunk 2 above)
Comment (sumeetyeola) [reply css/utilities.css:21]: Thanks for the suggestion. I will try a better one.

**Pull request merged**